src/sentry/south_migrations/0288_populate_release_project_new_groups.py

```diff
@@ -0,0 +1,721 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        "Write your forwards methods here."
+        db.commit_transaction()
+        for release in orm.Release.objects.all():
+          projects = release.projects.values_list('id', flat=True)
```

Comment:
Coerce this to a `list()`, otherwise we're going to to be executing the query twice. Once for the `len(projects)` check, and again when we iterate it.